src/test/java/salesforcetest/steps/OpportunityStep.java

```diff
@@ -81,11 +84,13 @@ public void changesAnOpportunitySOwnerWith(final String ownerType) {
      */
     @Then("the application should display an information message in Opportunity page with the format {string}")
     public void displaysAnInformationMessageInOpportunityPageWithTheFormat(final String message) {
-        ToastMessageSpan toastMessageSpan = new ToastMessageSpan();
-        String actualResult = toastMessageSpan.getToastMessage();
-        String expectedResult = ReplacerMessages.replaceChangeOwnerMessage(message, context.getOpportunities()
-                .get(FIRST_OPPORTUNITY).getName());
-        Assert.assertEquals(actualResult, expectedResult);
+        if (userExperience.equals(USER_EXPERIENCE_LIGHTNING)) {
+            ToastMessageSpan toastMessageSpan = new ToastMessageSpan();
+            String actualResult = toastMessageSpan.getToastMessage();
+            String expectedResult = ReplacerMessages.replaceChangeOwnerMessage(message, context.getOpportunities()
+                    .get(FIRST_OPPORTUNITY).getName());
+            Assert.assertEquals(actualResult, expectedResult);
+        }
```

Comment:
if the user experience is not Lightning, what happened?

Reply:
Classic user experience don't have a successful message

Reply:
But you need to validate something right?

Reply:
I have 2 validation, but 1 only for classic, and 2 for lightninh

Reply:
where is the classic UE validation?